cluster-soak/src/main/resources/common.conf

```diff
@@ -1,25 +1,72 @@
+dns-dispatcher {
+      type = "Dispatcher"
+      executor = "thread-pool-executor"
+      throughput = 1
+
+      thread-pool-executor {
+        fixed-pool-size = 64
+      }
+    }
+
 akka {
   loggers = ["akka.event.slf4j.Slf4jLogger"]
   loglevel = "DEBUG"
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
   actor.provider = cluster
 
   # Restart the pod if unable to join the cluster
-  cluster.shutdown-after-unsuccessful-join-seed-nodes = 30s
+  cluster.shutdown-after-unsuccessful-join-seed-nodes = 60s
   coordinated-shutdown.exit-jvm = on
 
   cluster {
     downing-provider-class = "com.lightbend.akka.sbr.SplitBrainResolverProvider"
   }
 
+  actor {
+    deployment {
+      "/IO-DNS/inet-address" {
+        mailbox = "unbounded"
+        router = "consistent-hashing-pool"
+        nr-of-instances = 8
+      }
+
+      "/IO-DNS/inet-address/*" {
+        dispatcher = "dns-dispatcher"
+      }
+    }
+  }
+
+
+  cluster {
+    log-info-verbose = off
+    debug {
+      # log heartbeat events (very verbose, useful mostly when debugging heartbeating issues)
+      verbose-heartbeat-logging = on
+
+      # log verbose details about gossip
+      verbose-gossip-logging = on
```

Comment:
this might be too verbose unless troubleshooting something specific?

Reply:
Turning all these off